### PR TITLE
Modify README.md file

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to
- Create class `Person` with  the following:
  - Instance vars: `@id`, `@name`, and `@age`.
  - Constructor with `name`,  `age`, and `parent_permission` as parameter. `name` and `parent_permission` are optional and have default values of `"Unknown"` and `true`.
  - Getters for `@id`, `@name`, and `@age`.
  - Setters for `@name` and `@age`.
  - Private method `is_of_age?` that returns `true` if `@age` is greater or equal to 18 and `false` otherwise.
  - Public method `can_use_services?` that returns `true` if person is of age or if they have permission from parents.
- Create class `Student` with the following:
  - Inherits from `Person`.
  - Constructor extends parent's constructor by adding `@classroom` and a parameter for it.
  - Method `play_hooky` that returns `"¯\(ツ)/¯"`.
- Create class `Teacher` with the following:
  - Inherits from `Person`.
  - Constructor extends parent's constructor by adding `@specialization` and a parameter for it.
  - Override `can_use_services?` so it always returns `true`.
- I saved each class in its file

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -3,12 +3,12 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(name = "Unknown", age = 0, parent_permission = true, classroom = "Unknown")
-    super(name, age, parent_permission)
+  def initialize(name = 'Unknown', classroom = 'Unknown', parent_permission: true)
+    super(name, parent_permission)
     @classroom = classroom
   end
```
Comment (thecodechaser) [inline student.rb]: - [ ] Please pass age in the `initializer` method and in the `super` method as parameter. So while creating students users should be able to provide their age.

File: teacher.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 class Teacher < Person
   attr_accessor :specialization
 
-  def initialize(name = "Unknown", age = 0, parent_permission = true, specialization = "Unknown")
-    super(name, age, parent_permission)
+  def initialize(name = 'Unknown', specialization = 'Unknown', parent_permission: true)
+    super(name, parent_permission)
     @specialization = specialization
   end
```
Comment (thecodechaser) [inline teacher.rb]: - [ ] Please pass age in the `initializer` method and in the `super` method as parameter. So while creating teachers users should be able to provide their age.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ksigei👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @ksigei 👋,

Great job adding age to the initializer of both the student and teacher classes/models.

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**